AFM: Docs for the April release (#29)
mulesoft/docs-functional-monitoring: modules/ROOT/pages/afm-view-monitor-history.adoc

= View the Details of Past Executions of a Test
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:page-aliases: afm-view-test-history.adoc

:imagesdir: ../assets/images

You can review the details of past executions of a monitor's test. The *Test History* page for the monitor lists executions of the test. The *Execution Detail* page that appears for a selected execution displays the pairs of HTTP REST methods and endpoints that were tested.

For example, the entries on *Execution Details* page look like these:

[%hardbreaks]
`DELETE "http://httpbin.org/delete"`
`GET "http://httpbin.org/get"`
`POST "http://httpbin.org/post"`

When you expand an entry, you can see the response and the request, as well as the result for each assertion.

== Procedure

> . In the list of your monitors, select the monitor that includes the test.

Reviewer: ```suggestion
. In the list of monitors, select the one that includes the test.
```

> . On the page that is opened, click the three dots next to the name of the monitor and select *View Test History*.

Reviewer: Suggest changing this in earlier versions as well:
```suggestion
. Click the three dots next to the name of the monitor and select *View Test History*.
```

> . On the *Test History* page, click the list entry for the test execution that you want to see the details of.

Reviewer: ditto: assume that the previous step locates the user where they should be, so we need not repeat:
```suggestion
. Click the list entry for the test execution that you want to see the details of.
```

+

> You can identify a test execution the following details in the list entries:

Reviewer: "by"?
```suggestion
You can identify a test execution by the following details in the list entries:
```

+
* Its sequential number

> * Its result: Passed or Failed

Reviewer: If these are in the UI, then *Passed* or *Failed*
If syntax, then `Passed` or `Failed`
If a general state, then passed or failed
